docs/UserGuide.md

```diff
@@ -668,37 +674,50 @@ Encountered an unfamiliar term when using LinkMeIn? Read the respective descript
+| `PENDING INTERVIEW`   | `PI`, `PENDINT`, `PENDINGINT`, `PENDINGINTERVIEW`, `P I`, `PENDING I`   | You have submitted the application, and are either waiting for an interview offer, or waiting to go for the interview.              |
+| `PENDING OUTCOME`     | `PO`, `PENDOUT`, `PENDINGOUT`, `PENDINGOUTCOME`, `P O`, `PENDING O`     | You have went through the interview, but the final decision or outcome (whether accepted or rejected) hasn't been communicated yet. |
+| `ACCEPTED`            | `A`, `ACC`, `ACPT`, `ACCEPT`                                            | You have been offered the internship position. Congratulations!                                                                     |
+| `REJECTED`            | `R`, `REJ`, `REJECT`                                                    | Unfortunately, the application wasn't successful and you were not offered the internship. Don't worry, try again next time!         |
+
+
+#### Email Format Description
+The email format is `local-part@domain`. 
 
-| Application Status  | Other Accepted Inputs                                        | Description                                                                                                                                         |
-|---------------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
-| PENDING APPLICATION | PA, PENDAPP, PENDINGAPP, PENDINGAPPLICATION, P A, PENDING A  | This means that you have not submitted the application, but are currently working on it.                                                            |
-| PENDING INTERVIEW   | PI, PENDINT, PENDINGINT, PENDINGINTERVIEW, P I, PENDING I    | This means that you have submitted the application, and are either waiting for an interview offer, or waiting to go for the interview.              |
-| PENDING OUTCOME     | PO, PENDOUT, PENDINGOUT, PENDINGOUTCOME, P O, PENDING O      | This means that you have went through the interview, but the final decision or outcome (whether accepted or rejected) hasn't been communicated yet. |
-| ACCEPTED            | A, ACC, ACPT, ACCEPT, ACCEPTED                               | This means you have been offered the internship position. Congratulations!                                                                          |
-| REJECTED            | R, REJ, REJECT, REJECTED                                     | Unfortunately, this means that the application wasn't successful and you were not offered the internship. Don't worry, try again next time!         |
+`local-part` constraints: 
+* Only contain alphanumeric characters and these special characters, excluding the parentheses, (+_.-). 
+* May not start or end with any special characters.
 
+`domain` name constraints:
+* Made up of domain labels separated by periods.
```

Comment:
Wait, I think our current check right now doesn't check for periods?

Reply:
We just copied over AB3 error message and implementation for the email constraints. Does it not work?